src/incubator/Dialog/index.tsx

```diff
@@ -1,11 +1,15 @@
 import React, {PropsWithChildren, useMemo, useCallback} from 'react';
 import {StyleSheet, StyleProp, ViewStyle} from 'react-native';
 import {useSharedValue, withTiming, useAnimatedStyle} from 'react-native-reanimated';
-import TransitionView, {TransitionViewDirection, TransitionViewAnimationType} from '../TransitionView/index';
-import PanView, {PanViewDirections} from '../panView/index';
+import TransitionView, {TransitionViewAnimationType} from '../TransitionView/index';
+import PanView from '../panView/index';
+import {PanningDirections, PanningDirectionsEnum} from '../panView/panningUtil';
```

Comment:
Such things should exported directly from PanView file so other component shouldn't be dependent on internal files of another component

Reply:
Done, tried to use the `globals` but the enum did not work so I used your original suggestion